src/environments/environment.ts

```diff
@@ -6,6 +6,6 @@
 export const environment = {
   production: false,
   test: false,
-  couchAddress: 'http://127.0.0.1:2200/',
+  couchAddress: 'http://192.168.0.56:2200/',
```

Comment:
looks like this is the wrong file commited

Reply:
Whoops...fixing now